eva_submission/submission_qc_checks.py

```diff
@@ -0,0 +1,202 @@
+            if accessioning_log_files:
+                # check if accessioning job completed successfullyy
+                if self.check_if_job_completed_successfully(accessioning_log_files[0]):
+                    logger.info(f'Accessioning completed successfully for study {project_accession} '
+                                f'and file {os.path.basename(accessioning_log_files[0])}')
+                else:
+                    logger.error(f'Accessioning failed for study {project_accession} '
+                                 f'and file {os.path.basename(accessioning_log_files[0])}')
+
+                    self.check_for_errors_in_case_of_job_failure(accessioning_log_files[0])
+
+                # check if any variants were skippped while accessioning
+                variants_skipped_line = self.check_if_variants_were_skipped(accessioning_log_files[0])
+                if variants_skipped_line:
+                    logger.error(f'Some of the variants were skipped while accessioning. '
+                                 f'The following line from logs describes the total variants that were skipped.'
+                                 f'\n{variants_skipped_line}')
+            else:
+                logger.error(f'No accessioning log file could be found for study {project_accession}')
+
+    def check_if_variant_load_completed_successfully(self, project_accession, browsable_files, path_to_data_dir):
+        for file in browsable_files:
+            pipeline_log_files = glob.glob(f"{path_to_data_dir}/00_logs/pipeline.*{file}*.log")
+            if pipeline_log_files:
+                # check if variant load job completed successfully
+                if self.check_if_job_completed_successfully(pipeline_log_files[0]):
+                    logger.info(f'Variant load stage completed successfully for study {project_accession} '
+                                f'and file {os.path.basename(pipeline_log_files[0])}')
+                else:
+                    logger.error(f'Variant load stage failed for study {project_accession} '
+                                 f'and file {os.path.basename(pipeline_log_files[0])}')
+
+                    self.check_for_errors_in_case_of_job_failure(pipeline_log_files[0])
+            else:
+                logger.error(f'No pipeline log file could be found for study {project_accession}')
+
+    def run_qc_checks_for_submission(self):
+        profile = cfg['maven']['environment']
+        private_config_xml_file = cfg['maven']['settings_file']
+        project_accession = self.eload_cfg.query('brokering', 'ena', 'PROJECT')
+        path_to_data_dir = Path(cfg['projects_dir'], project_accession)
+        taxonomy = self.eload_cfg.query('submission', 'taxonomy_id')
+        analyses = self.eload_cfg.query('brokering', 'analyses')
+
+        browsable_files = self.get_browsable_files_for_study(profile, private_config_xml_file, project_accession)
```

Comment:
- This assumes that the browsable files have been entered correctly in the database. 
I think you should get the file names from the config as well, then check that these files have been added to the browsable files.
These should be in the brokered files in `brokering.analyses.vcf_files`

Reply:
done